protocol/messenger.go

```diff
@@ -1660,6 +1660,16 @@ func (m *Messenger) SetInstallationMetadata(id string, data *multidevice.Install
 	return m.setInstallationMetadata(id, data)
 }
 
+func (m *Messenger) SetInstallationName(id string, name string) error {
+	installation, ok := m.allInstallations.Load(id)
+	if !ok {
+		return errors.New("no installation found")
+	}
+
+	installation.InstallationMetadata.Name = name
+	return m.encryptor.SetInstallationName(&m.identity.PublicKey, id, name)
```

Comment:
we should prefer use m.IdentityPublicKey() instead access field directly

Reply:
Well I just copied from `setInstallationMetadata` 😄 
https://github.com/status-im/status-go/blob/93cee626f06132e58eddc013b9adb229c6e95f8e/protocol/messenger.go#L1656

But no prob, will fix both 👌